src/RequestGenerator/DefaultRequestGenerator.php

```diff
@@ -92,7 +101,7 @@ public function getFrontendRequest(Request $request) {
    * @return $this
    */
   public function setFrontendPages(array $pages) {
-    $this->frontendPages = array_flip($pages);
```

Comment:
as mentioned in the wv-sat PR let's not change array $pages - let's keep the API as it is. no reason to introduce a breaking change here.

Reply:
I want to use it for passing through `'robots.txt' => '/' . getenv('SITE') . '/robots.txt',` for WV-4277

Reply:
but maybe that is better handled by nginx?

Reply:
will try out handling it in nginx config now